index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Home</title>
+  </head>
+  <body>
+    <header>
+      <div class="navbar">
+        <ul>
+          <li> <img
+              src="logo-image.png"
+              alt="logo image of techgirlshub"
+              class="logo" /></li>
+          <li><a href="#about-Us">About Us</a></li>
+          <li><a href="Services.html">Services</a></li>
+          <li><a href="projects.html">Projects</a></li>
+          <li><a href="#team-members">Team members</a></li>
+           <li><a href="feedbackRating.php">Feedback Rating</a></li>
+          <li><a href="#contact-Us">Contact Us</a></li>
```

Comment:
Add a link for contact us as contactUs.php

Reply:
and about us link